src/AzureADB2C/Provider.php

```diff
@@ -202,7 +202,7 @@ protected function mapUserToObject(array $user)
             'id'       => $user['sub'],
             'nickname' => $user['name'],
             'name'     => $user['name'],
-            'email'    => $user['emails'][0],
+            'email'    => $user['emails'][0] ?? $user['email'],
```

Comment:
If none of them are set, shall this fall back to `null` ?

Reply:
@lucasmichot Yeah, check out: https://3v4l.org/p3Fb4

```php
<?php

$user = [];

var_dump($user['emails'][0] ?? $user['email']); //returns NULL
```

Reply:
But this emits a warning. Lets do `$user['emails'][0] ?? $user['email'] ?? null`?